### PR TITLE
Added Amoco to convenience json file

### PR DESCRIPTION
**Pull request by lizwait**

### CODE REVIEW

File: brands/shop/convenience.json [hunk 1]
```diff
@@ -165,6 +165,15 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Amoco": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "Amoco",
+      "brand:wikidata": "Q465952",
+      "name": "AmeriStop",
```
Comment (bhousel) [inline brands/shop/convenience.json]: The `name` should also be "Amoco"..

File: brands/shop/convenience.json [hunk 2]
```diff
@@ -165,6 +165,15 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Amoco": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "Amoco",
+      "brand:wikidata": "Q465952",
```
Comment (bhousel) [inline brands/shop/convenience.json]: Can also add
`"brand:wikipedia": "en:Amoco"`

File: brands/shop/convenience.json [hunk 3]
```diff
@@ -639,6 +630,15 @@
       "shop": "convenience"
     }
   },
+  "shop/convenience|Huck's": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "Huck's Food & Fuel",
+      "brand:wikidata": "Q56276328",
+      "name": "Huck's Food & Fuel",
```
Comment (bhousel) [inline brands/shop/convenience.json]: Can also add
`"brand:wikipedia": "en:Huck's Food & Fuel",`

**Comment by lizwait** [comment]
Hey @bhousel! I believe I've added all of the changes. :) Thanks!

**Comment by bhousel** [comment]
Thanks @lizwait for making the fixes, and welcome! 👋

**Pull request merged**